File: Backend/uploads/factorial.c
// Paste your C code here to analyze it!
#include <stdio.h>

long calculateFactorial(int n) {
    if (n < 0) return -1;
    if (n == 0) return 1;
    
    long result = 1;
    for (int i = 1; i <= n; i++) {
        result *= i;
    }
    return result;
}

int main() {
    printf("%ld\n", calculateFactorial(5));
    return 0;
}

File: Backend/uploads/factorial.py
# Paste your Python code here to analyze it!
def calculate_factorial(n):
    if n < 0:
        return None
    elif n == 0:
        return 1
    else:
        # A simple recursive implementation
        result = 1
        for i in range(1, n + 1):
            result *= i
        return result

print(calculate_factorial(5))
